signer/tuf_on_ci_sign/_user.py

```diff
@@ -0,0 +1,75 @@
+import sys
+from configparser import ConfigParser
+
+import click
+from securesystemslib.signer import Key, Signer
+
+
+def bold(text: str) -> str:
+    return click.style(text, bold=True)
+
+
+class User:
+    """Class that manages user configuration and manages the users signer cache"""
+
+    def __init__(self, path: str):
+        self._config_path = path
+
+        self._config = ConfigParser(interpolation=None)
```

Comment:
the argument is added because
* we don't need the fancy interpolation feature of ConfigParser
* interpolation prevents using some characters in config values and we want to use them in priv key uris